docs-src/app-dev-context/developing-activities.md

```diff
@@ -10,3 +10,5 @@ tags:
 One of the primary things that Workflows do is orchestrate the execution of Activities.
 Activities are normal function/method executions that can interact with the world.
 For the Workflow to be able to execute the Activity, we must define the [Activity Definition](/concepts/what-is-an-activity-definition).
+
+Temporal requires that Activities are idempotent. Activities are idempotent if multiple applications of that operation do not change the state of the system beyond the initial application.
```

Comment:
Like to the Idempotency section in the concept?

Reply:
I'll point out that this wording says that Temporal _requires_ Activities to be idempotent, but the wording in `docs-src/concepts/what-is-an-activity.md` says that Temporal _recommends_ them to be idempotent. 

Personally, I think that recommends (or perhaps "strongly recommends") is better wording, from the perspective that Temporal can do its job whether or not the Activity is idempotent (in contrast to whether or not a Workflow is deterministic, a violation of which can lead to failure). Others have tried to make the case that "required" is better. 

I will leave it up to the Docs team to decide which is more accurate, but I do suggest that Activity idempotence will confuse people if it's described as "recommended" in one place and "required" in another.

Reply:
It doesn't require, and there are times when idempotency isn't possible / supported by a downstream (possibly 3rd party) API

Reply:
🤷‍♂️ I guess "in order for your application to be correct / always function as intended in the presence of failure, activities must be idempotent" can be shortened to "activities must be idempotent"